apps/web/components/Celebration.tsx

```diff
@@ -4,17 +4,44 @@ import Image from "next/image";
 import { getCelebrationImage } from "@/utils/celebration";
 import { Button } from "@/components/Button";
 
-export function Celebration(props: { message: string }) {
+export function Celebration(props: { type: string }) {
```

Comment:
Thanks. I'd just use the existing format and pass in the relevant message. No need to change the props. Or this component at all.